packages/server/lib/controllers/activity.controller.ts

```diff
@@ -30,7 +30,11 @@ class ActivityController {
 
     public async getMessages(req: Request, res: Response, next: NextFunction) {
         try {
-            const logIds = req.query['logIds'] ? (req.query['logIds'] as string).split(',').map((logId) => parseInt(logId)) : [];
+            const logIds: number[] = req.query['logIds'] ? (req.query['logIds'] as string).split(',').map((logId) => parseInt(logId)) : [];
```

Comment:
can we not to optional chaining rather than the ternery operator: `(req.query['logIds'] as string)?.split(',').map((logId) => parseInt(logId))`

Reply:
actually we should add same type of params validation as there is in the others endpoint, will change that

Reply:
Fixed [f944701](https://github.com/NangoHQ/nango/pull/1665/commits/f94470197db8fa6650ca29a4082f1794cbe8ff77)